scripts/testdata/satellite-config.yaml.lock

```diff
@@ -187,15 +187,18 @@ kademlia.operator.wallet: ""
 # a node's ratio of being up/online vs. down/offline
 # overlay.node.uptime-ratio: 0.9
 
-# how frequently checker should audit segments
-# repairer.interval: 1h0m0s
+# how frequently irrepair checker should audit segments
+# repairer.irrepair-interval: 30m0s
```

Comment:
Just trying to check everything irreparable every 30 minutes seems like it would reach diminishing returns at some point... If I've checked something every 30 minutes for a year (17.5k times!)... I never give up?  I never back off?  Is there another ticket for this scope?

Reply:
I took 30min, only to wanted to run irreparable task more frequently than repairable. The time is configurable.